.github/workflows/build-publish-image.yml

```diff
@@ -41,14 +38,14 @@ jobs:
       uses: docker/build-push-action@v5
       with:
         context: .
-        file: ./docker/Dockerfile-py${{ matrix.python-version }}
+        file: ./docker/Dockerfile
         push: true
-        tags: ghcr.io/${{ github.repository }}/bandit:py${{ matrix.python-version }}-${{ matrix.architecture }}
-        platforms: linux/${{ matrix.architecture }}
+        tags: ghcr.io/${{ github.repository }}/bandit:latest
```

Comment:
I also don't see us including the version of Bandit here but maybe that's because we are using the same action on a schedule and for releases?

Reply:
Oh, also, if we're doing this on a schedule like this, does that mean we're publishing the head of `main` every week? Do we want to ensure we're only publishing the last released version with the new image?

Reply:
I also thought it best to not use bandit releases, latest should always be usable.

Reply:
Getting back on to this , apologies for the haitus. 

> If we're running a schedule, could we find the latest tag, check that out and rebuild? I assume the scheduled rebuild is to address vulnerabilities in the base image like I raised as a concern?

Yes, this rebuild the entire image every sunday at midnight. This covers off any CVEs in the base image that have collected since our last release.

I am going to push another commit now with a workflow_dispatch. this will allow us to manually start the action if anything particular comes out nasty between the scheduled runs.